build/media-suite_helper.sh

```diff
@@ -1216,13 +1216,14 @@ log() {
 
 create_build_dir() {
     local build_dir="build${1:+-$1}-$bits"
-    if [[ "$(basename "$(pwd)")" = "$build_dir" ]]; then
-        rm -rf ./*
-    elif [[ -d "$build_dir" ]] && ! rm -rf ./"$build_dir"; then
-        cd_safe "$build_dir" && rm -rf ./*
-    else
-        mkdir "$build_dir" && cd_safe "$build_dir"
+    if [[ ! -f "$(get_first_subdir)/do_not_clean" ]]; then
+        if [[ "$(basename "$(pwd)")" = "$build_dir" ]]; then
+            rm -rf ./*
```

Comment:
You should do like below and `cd_safe ..`, otherwise you'll be creating another build_dir inside build_dir.

Reply:
fixed